### ISSUE TITLE
[📦]: Regions Unexplored

### ISSUE THREAD

**Issue by QwertRvg** [MehVahdJukaar/WoodGood#383]
### Before Continuing:

- [X] See [EC Backlog](https://github.com/users/MehVahdJukaar/projects/1/views/1) if someone already suggested a mod that you are going to suggest.

### Curseforge (Normal)

https://www.curseforge.com/minecraft/mc-mods/regions-unexplored

### FABRIC Version Availability

- [ ] 1.18.2
- [X] 1.19.2
- [X] 1.20.1

### FORGE Version Availability

- [X] 1.18.2
- [X] 1.19.2
- [X] 1.20.1

### MODID

regions_unexplored

### Types

**Block:**
- Branch (with variants for all Vanilla and RU wood types)

**Item:**
- woodtype_branch

### Optional Details

The branches all use the texture of the log blocks, so that is automatic.
I'm not super knowledgeable with this stuff, but the item textures for the branches seem more "custom", so not sure if they are easily automated.
Also the Joshua and Palm trees from RU use "Beards" instead of branches.

**Comment by dadadadaan** [comment]
Is there any potential to get the leaves from Regions Unexplored to have Chipped mod variants, like flowering, frosted, etc.?

**Comment by Xelbayria** [comment]
Ok. IIRC, the beard from Regions Unexplored (RU) doesn't have vanilla variant (oak, birch, so on). Only WoodType on RU's side like "joshua" and others. 

The Details are in `#795` above. 

> Is there any potential to get the leaves from Regions Unexplored to have Chipped mod variants, like flowering, frosted, etc.?

Do you meant it like this: 

RU (Decorative Blocks) + Chipped's LeavesType?

**Comment by dadadadaan** [comment]
Yes, I believe that's correct. I can put vanilla leaves in the Chipped workbenches but not the leaves from the RU trees. Sorry for not doing great with the phrasing there😅

**Comment by dadadadaan** [comment]
I also noticed the chipped workbenches don't have the styles of door and trapdoor available as options for the vanilla woods (i.e. RU's pine door shape isn't an option for Vanilla dark oak doors).

**Comment by Xelbayria** [comment]
alright, i'll take a look at the RU's leaves (decorative blocks). You say Chipped don't have door variants? I'm 100% sure that EC's Chipped-Module support doors and also trapdoors. 

unless, it's named like `regionsunexplored:pine_door`. the door variant from RU's is not supported blc that is a the vanilla door.

**Comment by dadadadaan** [comment]
Ahhh I understand. I figured it would be tough to systematically recognize a door or trapdoor pattern from a mod to then implant it in another mod's ecosystem
